include/fg/defines.h

```diff
@@ -9,7 +9,7 @@
 
 #pragma once
 
-#ifdef OS_WIN
+#if OS_WIN==1
```

Comment:
What is the reasoning behind this?

Reply:
you should just use ifdef instead of this. Its more readable.

Reply:
They don't work reliabily, sometimes(platforms?) they bailed out. Which is why I made the switch to this one. I will give it another try.